Gruntfile.js

```diff
@@ -86,44 +82,38 @@ module.exports = function (grunt) {
       }
     },
     jshint: {
-      browser: {
-        src: clientSideJS,
-        options: browserJSHint
-      },
-      node: {
-        src: nodeJS,
-        options: nodeJSHint
-      }
+      src: allJS,
+      options: JSHint
     },
     gettext_finder: {
-      files: ["views/*.html", "views/**/*.html"],
+      files: ['views/*.html', 'views/**/*.html'],
       options: {
-        pathToJSON: ["locale/en_US/webmaker.org.json",
-                     "locale/en_US/localweb.json"
-                    ],
-        ignoreKeys: grunt.file.readJSON("gtf-ignored-keys.json")
-      },
+        pathToJSON: ['locale/en_US/webmaker.org.json',
+          'locale/en_US/localweb.json'
+        ],
+        ignoreKeys: grunt.file.readJSON('gtf-ignored-keys.json')
+      }
     },
     angular_i18n_finder: {
-      files: ["public/views/*.html", "public/views/**/*.html"],
+      files: ['public/views/*.html', 'public/views/**/*.html'],
       options: {
-        format: "chromeI18n",
-        pathToJSON: ["locale/en_US/appmaker.json",
-                      "locale/en_US/badges.json",
-                      "locale/en_US/explore.json",
-                      "locale/en_US/feedback.json",
-                      "locale/en_US/make-your-own.json",
-                      "locale/en_US/mentor.json",
-                      "locale/en_US/music-video.json",
-                      "locale/en_US/private-eye.json",
-                      "locale/en_US/remix-your-school.json",
-                      "locale/en_US/resources.json",
-                      "locale/en_US/signup.json",
-                      "locale/en_US/tools.json",
-                      "locale/en_US/user-box.json"
-                    ],
-        ignoreKeys: grunt.file.readJSON("angular-i18n-ignoreKeys.json")
-      },
+        format: 'chromeI18n',
+        pathToJSON: ['locale/en_US/appmaker.json',
```

Comment:
So while this was a jsbeautify change, it has no prob with moving it to a new line, so fixing this!